index.bs

```diff
@@ -99,8 +99,31 @@ Usage Example {#example}
     observer.observe({type: "longtask", buffered: true});
     // Long script execution after this will result in queueing
     // and receiving "longtask" entries in the observer.
+
+    // Register observer for previous and future long animation frame notifications.
+    // After this, long periods where the main thread is busy will result in queueing
+    // and receiving "long-animation-frame" entries in the observer.
+    observer.observe({type: "long-animation-frame", buffered: true});
 </pre>
 
+Long Animation Frames vs. Long Tasks {#loaf-vs-longtasks}
+---------------------------------------------------------
+
+While both long tasks and long animation frames measure congestion and jank, long animation frames
+provide information that has a better correlation with how user preceive this type of congestion.
+That's because long animation frames measure a sequence that begins when the main thread is idle,
+and end when the frame either renders or the user agents decides there is nothing to render.
+
+The [=task=] term is somewhat of an implementation detail, and the long animation frame addition
+attempts to remedy that by introducing a more user-centric metric of the same phenomenon of main
+thread congestion/jank.
+
+Because long animation frames are guaranteed to have a maximum of one rendering phase, we can also
+use them to expose additional information about the rendering phase itself, such as
+{{PerformanceLongAnimationFrameTiming/renderStart}} and {{PerformanceLongAnimationFrameTiming/styleAndLayoutStart}}.
+
+For a detailed explanation about long animation frames, see the <a href="https://github.com/w3c/longtasks/blob/main/loaf-explainer.md">explainer</a>.
```

Comment:
For now that's fine, but at some point I think we'd want to incorporate the explanations here.